### PR TITLE
feat(datepicker) add albanian locale

### LINKED ISSUE #5541: Albanian Language needed
Please add the Albanian Language (sq) to this project!

### PR DESCRIPTION
**Pull request by agoncecelia**
# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.

closes #5541

### CODE REVIEW

**Comment by daniloff200** [comment]
well, I see some tests are failed here.
you can take this PR, as an example, here they are passing https://github.com/valor-software/ngx-bootstrap/pull/5556/files

**Comment by agoncecelia** [comment]
@daniloff200 I saw that tests are failing but couldn't figure them out, thought that someone could help me out with the tests. I can give it another try someday when I have time :).

File: src/chronos/i18n/sq.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  dayOfMonthOrdinalParse: /\d{1,2}(st|nd|rd|th)/,
+  ordinal(_num: number): string {
+    const num = Number(_num);
+    const b = num % 10,
+      output = (~~(num % 100 / 10) === 1) ? 'i' :
+        (b === 1) ? 'i' :
+          (b === 2) ? 'i' :
+            (b === 3) ? 'i' : 'i';
+    return num + output;
```
Comment (IraErshova) [inline src/chronos/i18n/sq.ts]: Is this code correct? You can change it to return `num + 'i'`, cause you always use only `'i'`. I don't know Albanian so could you please help me? What is the correct rule to create an ordinal number 1 (for example):
`i1` or` i 1` or `1i` (your case)

**Comment by IraErshova** [comment]
@agoncecelia Hi! I'm working on your PR, trying to help you fix it. Could you please check my comment, I need information related to Albanian?

File: src/chronos/i18n/sq.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    h : 'një orë',
+    hh : '%d orë',
+    d : 'një ditë',
+    dd : '%d ditë',
+    M : 'një muaj',
+    MM : '%d muaj',
+    y : 'një vit',
+    yy : '%d vite'
+  },
+  dayOfMonthOrdinalParse: /\d{1,2}(st|nd|rd|th)/,
```
Comment (IraErshova) [inline src/chronos/i18n/sq.ts]: This regular expression is wrong. It depends on how ordinal number looks like in Albanian. Look at my example for number 1. Which one is correct? Could you help me, please?
- `i1`
- `i 1`
- `1i`

Comment (IraErshova) [reply src/chronos/i18n/sq.ts]: I checked moment.js code I saw this :
```
dayOfMonthOrdinalParse: /\d{1,2}\./,
ordinal : '%d.',
```
Is this correct? They add `.` symbol at the end of the number

Comment (agoncecelia) [reply src/chronos/i18n/sq.ts]: Hi @IraErshova, yes we can put . at the end. Seems now that all tests are passing and everything looks good. Thank you very much!

Comment (IraErshova) [reply src/chronos/i18n/sq.ts]: @agoncecelia Thanks for the reply. yes, I fixed your test. you are welcome :)

**Review by daniloff200 — DISMISSED** [review]

**Review by IraErshova — APPROVED** [review]

**Review by daniloff200 — APPROVED** [review]

**Comment by dmitry-zhemchugov** [comment]
Fixed e2e tests for the 'sq' locale. 
Tested. Ready to merge

**Comment by agoncecelia** [comment]
Thank you everyone for all your help :)!

**Pull request merged**